apps/web/src/components/settings/OrganizationMembersSection.tsx

```
@@ -0,0 +1,241 @@
+        // since navigated away from.
+        //
+        // Switching organizations left the PREVIOUS org's roster on screen
+        // until the new fetch resolved — so for a second or two the panel
+        // showed real people who are not in the org named above them, with a
+        // Remove button next to each. Acting on that stale list removes
+        // somebody from an org you are not looking at.
+        //
+        // The `cancelled` flag also handles the out-of-order case: switch A→B→A
+        // quickly and B's slower response would otherwise land last and win.
+        let cancelled = false;
+        setLoading(true);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/apps/web/src/components/settings/OrganizationMembersSection.tsx:64:9
  62 |         // quickly and B's slower response would otherwise land last and win.
  63 |         let cancelled = false;
> 64 |         setLoading(true);
     |         ^^^^^^^^^^ Avoid calling setState() directly within an effect
  65 |         setMembers([]);
  66 |         setInvitations([]);
  67 |

(react-hooks/set-state-in-effect)